src/script/script.go

```diff
@@ -70,7 +70,7 @@ func RunCommand(cmd ...string) error {
 	if runtime.GOOS == "windows" {
 		cmd = append([]string{"cmd", "/C"}, cmd...)
 	}
-	subProcess := exec.Command(cmd[0], cmd[1:]...)
+	subProcess := exec.Command(cmd[0], cmd[1:]...) // #nosec
```

Comment:
This was necessary to get the linter to pass. Not sure why this wasn't an issue before.